### ISSUE TITLE
TypeError: the JSON object must be str, not 'bytes'

### ISSUE THREAD

**Issue by T-002** [tonybaloney/sphinxcontrib-confluencebuilder#55]
Hi,

I am trying to build my sphinx documentation with your builder. Unfortunately it fails, which is my motivation for posting this issue.

### Issue Details
I called the builder using
`sphinx-build -E -b confluence -d documentation/doctrees doc_source documentation/confluence`

The corresponding error log contains the following data:
```
# Sphinx version: 1.6.2
# Python version: 3.5.2 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.6
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "[...]/sphinx/cmdline.py", line 305, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "[...]/sphinx/application.py", line 234, in __init__
    self._init_builder()
  File "[...]/sphinx/application.py", line 311, in _init_builder
    self.builder.init()
  File "[...]/sphinxcontrib/confluencebuilder/builder.py", line 102, in init
    self.publisher.connect()
  File [...]/sphinxcontrib/confluencebuilder/publisher.py", line 62, in connect
    'limit': 1
  File "[...]/sphinxcontrib/confluencebuilder/rest.py", line 54, in get
    json_data = json.loads(rsp.content)
  File "/usr/local/Cellar/python3/3.5.2_1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
```

### Personal Fix
In order to fix the issue, I modified 
> [...]/sphinxcontrib/confluencebuilder/rest.py:54

 and updated it to
`            json_data = json.loads(rsp.content.decode("utf-8"))`

**Comment by jdknight** [comment]
@T-002, thanks for the information.

This should already be fixed on `master` (see #32; and also fa206b18982ad7765fd23e57b47718c6795df5ac). Once #53 is merged into master after a review, we'll close up all the active issues in milestone 0.7 and push out a release.

**Comment by T-002** [comment]
Thanks a lot. :)
I just installed it via pip ;)
